docs/_static/vdr/v2.yaml

```diff
@@ -0,0 +1,267 @@
+
+        error returns:
+        * 400 - Returned in case of malformed DID or service
+        * 404 - Corresponding DID document could not be found
+        * 500 - An error occurred while processing the request
+      tags:
+        - DID
+      operationId: updateService
+      parameters:
+          - name: service
+            in: body
+            description: Service to be added to the DID document.
+            required: true
+            schema:
+                $ref: '#/components/schemas/Service'
+      responses:
+        "200":
+          description: "Service has been updated successfully. Returns the service."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Service'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vdr/v2/did/{did}/verificationmethod:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example: did:web:example.com:iam:013c6fda-73e8-45ee-9220-48652dba854b
+        schema:
+          type: string
+    post:
+      summary: Creates and adds a new verificationMethod to the DID document.
+      description: |
+        It creates a new private public keypair. The public key is wrapped in verificationMethod. This method is added to the DID Document.
+        The key pair is used for all verificationMethods.
+
+        error returns:
+        * 404 - Corresponding DID document could not be found
+        * 500 - An error occurred while processing the request
+      operationId: addVerificationMethod
+      tags:
+        - DID
+      responses:
+        "200":
+          description: "New verification method has been created and added successfully. Returns the verification method."
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/VerificationMethod'
+        default:
+          $ref: '../common/error_response.yaml'
+  /internal/vdr/v2/did/{did}/verificationmethod/{kid}:
+    parameters:
+      - name: did
+        in: path
+        description: URL encoded DID.
+        required: true
+        example: did:web:example.com:iam:013c6fda-73e8-45ee-9220-48652dba854b
+        schema:
+          type: string
+      - name: kid
+        in: path
+        description: URL encoded kid identifying the verification method.
+        required: true
+        example: "did:web:example.com:iam:013c6fda-73e8-45ee-9220-48652dba854b#3106f751-59e3-440f-b57b-39a96a2da6c6"
+        schema:
+          type: string
+    delete:
+      summary: Delete a specific verification method
+      description: |
+        Removes the verification method from the DID Document.
+        Revokes the public key with the corresponding key-id.
+        Note: Other verification methods with different key-ids with the same private key will still be valid.
+
```

Comment:
Private keys aren't shared over verification methods (since each time, a new key pair is generated), so this a useless clause.
```suggestion

```